src/libs/antares/study/fwd.cpp

```diff
@@ -400,5 +400,62 @@ const char* DayAheadReserveManagementModeToCString(DayAheadReserveManagement daR
     return "";
 }
 
+
+std::string mpsExportStatusToString(const mpsExportStatus& mps_export_status)
```

Comment:
```suggestion
std::pair<bool, std::string> mpsExportStatusToString(const mpsExportStatus& mps_export_status)
```

Reply:
Could you be more specific ?

Reply:
The UNKNOWN enum value is only required to return an error in case the conversion wasn't successful. We can eliminate this value using an error-handling mechanism

Reply:
See also std::optional

Reply:
**mpsExportStatusToString** converts the status to a string, and its return value is based on its argument, which is necessarily in the range of values of the enum. So I guess we don't need this default returned value : it is useless.
We can either leave it as is, or return an empty string. What do you think ?